threaded_io_test.cc

```diff
@@ -102,6 +109,25 @@ int main(int argc, char* argv[]) {
   } else {
     outFactory = outputerFactoryGenerator(outputerName, "");
   }
+  char *p = getenv("HEP_MAX_BATCH_SIZE");
+  if ( p != NULL ) {
+    max_batch_size = atoi(p);
+  } else {
+    max_batch_size = 2;
+  }
+  p = getenv("HEP_IO_TYPE");
+  if ( p != NULL ) {
+    hdf_method = atoi(p);
+  } else {
+    hdf_method = 1;
+  }
+
+  if(argc > 7) {
+    max_batch_size = atoi(argv[7]);
+  }
+  if(argc > 8) {
+    hdf_method = atoi(argv[8]);
+  }
```

Comment:
This information should be part of the HDFOutputer configuration, not the configuration of the job itself since other Outputers do not need this info.